### PR TITLE
Rd 11197 add kafkajs support

### PR DESCRIPTION
**Pull request by therightstuff**

### CODE REVIEW

File: src/instrumentations/kafkajs/KafkaJsInstrumentation.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            getSpanAttributeMaxLength()
+          )
+        );
+      },
+      consumerHook: (span: Span, topic: string, message: Message) => {
+        span.setAttribute(
+          'messaging.consume.body',
+          CommonUtils.payloadStringify(
+            message.value.toString(),
+            ScrubContext.HTTP_REQUEST_QUERY,
```
Comment (saartochner) [inline src/instrumentations/kafkajs/KafkaJsInstrumentation.ts:30]: HTTP_RESPONSE_QUERY?

**Review by saartochner — APPROVED** [review]
LGTM 👍

File: src/instrumentations/kafkajs/KafkaJsInstrumentation.ts (hunk 1 above)
Comment (therightstuff) [reply src/instrumentations/kafkajs/KafkaJsInstrumentation.ts]: done, both for amqplib and kafkajs

**Pull request merged**